src/msgpack/MsgPackContext.cs

```diff
@@ -43,11 +59,16 @@ public void RegisterConverter<T>(IMsgPackConverter<T> converter)
         public IMsgPackConverter<T> GetConverter<T>()
         {
             var type = typeof(T);
-            return (IMsgPackConverter<T>) (GetConverterFromCache(type)
+            return (IMsgPackConverter<T>)(GetConverterFromCache(type)
                 ?? TryGenerateArrayConverter(type)
                 ?? TryGenerateMapConverter(type));
         }
```

Comment:
Please, add method `TryGenerateNullableConverter` for other nullable structures.